packages/mux-player/src/media-chrome/dialog.ts

```diff
@@ -90,7 +90,22 @@ class MediaDialog extends HTMLElement {
 
   show() {
     this.setAttribute("open", "");
-    focus(this);
+    if (document.activeElement?.nodeName.toLowerCase() === "mux-player") {
```

Comment:
the dialog shouldn't have to know about `mux-player` I think, thinking of when this would be included in media-chrome for example.

this code is based on the native dialog element and that assumes that the parent is the whole `document` and it's okay to steal focus because the dialog would go over the whole page. 

maybe it makes more sense we provide a "parent" for the dialog somehow

Reply:
cool yeah let's put this "on hold" until Tuesday to not rush things. I think a "fine for now" and "fast follow" should/would be fine to get alpha buttoned up, even if we don't think the implementation is what we'd want long term. That said, there's more work to be done here, regardless.